src/tracer.js

```diff
@@ -172,6 +166,23 @@ export const performStepFunctionLogic = (handlerReturnValue) => {
   );
 };
 
+const originalUnhandledRejection = process._events.unhandledRejection;
+export const hookUnhandledRejection = async (functionSpan) => {
+  process._events.unhandledRejection = async (reason, promise) => {
+    process._events.unhandledRejection = originalUnhandledRejection;
+    const err = Error(reason);
```

Comment:
consider creating our own Error something like
`export class TracerError extends Error {
  errorInfo?: string;
  errorType: ErrorType;
  message: string;
  stack?: any;
  constructor({ message, errorInfo, errorType }: ErrorInput) {
    let stringified = JSON.stringify({
      message,
      errorInfo,
      name: errorType,
    });
    super(stringified);
    this.message = message;
    this.errorInfo = errorInfo;
    this.errorType = errorType;
    this.name = errorType;
  }
}`

Reply:
Will do in the next PR :)